src/styles/index.ts

```diff
@@ -4025,6 +4025,17 @@ const styles = (theme: ThemeColors) =>
             verticalAlign: 'middle',
         },
 
+        chatBubbleCell: {
+            width: 24,
```

Comment:
I haven't looked at screenshots yet, but from Figma, I am using a 20x20 wrapper:
![CleanShot 2025-03-19 at 18 11 31@2x](https://github.com/user-attachments/assets/8882a299-a195-4141-aefb-45a2c3fb2876)

Then inside of that wrapper is our chat bubble icon, and here is a new svg you can use since I have it shifted down and resized just a tad to better hold the number: [chatbubble-counter.svg.zip](https://github.com/user-attachments/files/19352941/chatbubble-counter.svg.zip)

Reply:
Then on mobile views, we scale it down to 16x16:
![CleanShot 2025-03-19 at 18 16 00@2x](https://github.com/user-attachments/assets/a1d20407-354b-4a21-9ddf-57b8fa7b3297)

And we make the font size 7, which would be a new font var in our system.